.github/workflows/reusable-wait-for-npm-version-to-be-published.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  workflow_call:
+    inputs:
+      npm-package:
+        required: true
+        type: string
+      npm-version: # only exact versions allowed, not tags like "latest" or "alpha"
+        required: true
+        type: string
+      wait-minutes:
+        required: false
+        type: string
+        default: "10"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  wait-for-npm-version-to-be-published:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v4
+      - name: Wait
+        run: |
+          echo "Checking if ${{ inputs.npm-package }}@${{ inputs.npm-version }} is available on NPM (waiting max ${{ inputs.wait-minutes }} min)..."
```

Comment:
The code looks good, although the GHA syntax `${{ ... }}` makes it slightly harder to review.

I am wondering if it might not be better to have it in a separate bash file.

That would make it easier to maintain (reviews, QA, etc.).

Reply:
You're right, I created a separate bash file. Thanks